feature-utils/poly-look/src/react-components/filterChips.jsx

```diff
@@ -61,13 +61,26 @@ const FilterChips = ({
         )
   );
 
-  const [initialDefaultChips, setInitialDefaultChips] =
-    useState(defaultActiveChips);
+  const isFirstRender = useRef(true);
+  const initialActiveChips = useRef(defaultActiveChips);
 
-  if (defaultActiveChips !== initialDefaultChips) {
-    setActiveChips(defaultActiveChips);
-    setInitialDefaultChips(defaultActiveChips);
-  }
+  useEffect(() => {
+    if (isFirstRender.current) {
+      isFirstRender.current = false;
+      return;
+    }
+    if (
+      !(
+        initialActiveChips.current.length === defaultActiveChips.length &&
+        initialActiveChips.current.every(
+          (value, index) => value === defaultActiveChips[index]
+        )
```

Comment:
Note that that's not doing a deep equals, so it's still a bit error prone, but if it works it works.

Reply:
Ahm, true. I will use stringify.